src/wallet/rpcwallet.cpp

```diff
@@ -3257,6 +3262,9 @@ UniValue bumpfee(const JSONRPCRequest& request)
                 throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid estimate_mode parameter");
             }
         }
+        if (options.exists("reduce_output")) {
+            reduce_output = options["reduce_output"].get_int64();
+        }
```

Comment:
This will segfault on OOB

Reply:
Oops. Added bounds check in `feebumper::CreateTransaction`.

Reply:
Where?

Reply:
Also add `assert_raises_rpc_error` tests for:
 - when not a number;
 - when reduce_output < 0 and >= len(vout).

For instance, see 7f67dd0aa.

Reply:
No idea how that happened, but my changes are gone. Fixing, again.